eureka-core/src/main/java/com/netflix/eureka/registry/PeerAwareInstanceRegistryImpl.java

```diff
@@ -478,6 +486,11 @@ public boolean isLeaseExpirationEnabled() {
         return numberOfRenewsPerMinThreshold > 0 && getNumOfRenewsInLastMin() > numberOfRenewsPerMinThreshold;
     }
 
+    @com.netflix.servo.annotations.Monitor(name = METRIC_REGISTRY_PREFIX + "isLeaseExpirationEnabled", type = DataSourceType.GAUGE)
```

Comment:
Is servo a required dependency for eureka client?

Reply:
Yeah, it has been for some time.

```
$ git blame ./eureka-client/build.gradle | grep servo
f284e3e01 (David Liu          2015-07-24 13:33:32 -0700 13)     compile "com.netflix.servo:servo-core:${servoVersion}"
```